RemoteDispatch.csproj

```diff
@@ -1,30 +1,35 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net48</TargetFramework>
     <LangVersion>8.0</LangVersion>
     <Nullable>enable</Nullable>
     <RootNamespace>DvMod.RemoteDispatch</RootNamespace>
-    <InformationalVersion>0.5.1</InformationalVersion>
+    <Version>0.5.1</Version>
```

Comment:
InformationalVersion is the value shown in the Windows Explorer Properties Details view. Version is not visible anywhere.